lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_handlers.cpp

```diff
@@ -1655,12 +1665,12 @@ int sgw_s8_handle_delete_bearer_request(
 }
 
 // Handle NW-initiated dedicated bearer dectivation response from MME
-status_code_e sgw_s8_handle_s11_delete_bearer_response(
+int sgw_s8_handle_s11_delete_bearer_response(
     sgw_state_t* sgw_state,
     const itti_s11_nw_init_deactv_bearer_rsp_t* const
         s11_delete_bearer_response_p,
     imsi64_t imsi64) {
-  uint32_t rc = RETURNok;
```

Comment:
If these type changes are not necessary we should prefer to specify the size explicitly, as `int` may vary depending on the compiler and architecture. Are these changes needed?

Reply:
Thanks for bringing into notice.
Actually I made mistake to return an int. It should be status_code_e because in unit test case we validate the procedure based on return value.
https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/test/sgw_s8_task/sgw_dedicated_bearer.cpp#L199